src/models/thelio-spark-b1-n2/repairs.md

```diff
@@ -0,0 +1,630 @@
+
+3. Pull the top and bottom edges of the fan's plastic molding away from the heatsink it's clipped onto; the fan can be pulled away from the heatsink once the plastic molding's tabs are held far enough away from the heatsink.
+4. Repeat the process for the bottom clip, then pull the fan and clips away from the CPU heatsink.
+5. When reinstalling the CPU heatsink fan, the side with a stationary cover should face the heatsink, while the spinning side should face the front of the chassis.
+    - The cable should point towards the bottom inner corner (bottom-left when viewed from the back of the case.)
+
+## Replacing the CPU cooler and CPU:
+
+The CPU cooler dissipates heat from the CPU to the heatsink, where the CPU fans expel it from the system. Depending on your climate and the age of the machine, replacing the thermal paste between the CPU and the cooler/heatsink may help the system run cooler.
+
+The CPU uses an Intel LGA 1700 socket. The CPU cooler is an [Arctic Freezer 7 X Rev 3](https://www.arctic.de/us/Freezer-7-X/ACFRE00077A#specs).
+
+**Tools required:** Cross-head (Phillips) screwdriver, thermal paste  
+**Time estimate:** 45 minutes  
+**Difficulty:** High <span style="color:red;">●</span>
+
+### Steps to remove the CPU cooler/thermal paste:
+
+1. Follow the steps above to [remove the top case](#removing-the-top-case), [remove the side brace](#removing-the-side-brace), [remove the CPU duct](#removing-the-cpu-duct), and [remove the GPU](#replacing-the-gpu).
+    - Unplug the [heatsink-mounted CPU fan](#steps-to-replace-the-heatsink-mounted-cpu-fan) unless it's already been removed.
+2. While holding the CPU cooler in place so it doesn't fall, loosen the screws on the top and bottom of the CPU cooler holding it onto the mounting bracket.
+    - If you unscrew the screws too far, the 90-degree clips will detatch from the vertical mounting bar. Reattach them facing inwards when reinstalling the cooler.
+
+![CPU cooler screws](./img/cpu-cooler-screws.webp)
+
+3. Pull the cooler away from the CPU, alternating between the top and bottom to free the 90-degree clips from the mounting bracket.
+4. Using a paper towel, clean the existing thermal paste off of the heatsink and CPU. You may also use a small amount of rubbing alcohol if the old paste is dried or difficult to remove.
+
+![Thermal paste removal](./img/thermal-paste-removal.webp)
+
+### Steps to replace the CPU:
+
+1. Place the computer on its side so the motherboard is facing up.
+2. [Remove the RAM](#replacing-the-ram) to provide easier access to the cooler mounting bracket's plastic retention pins.
+3. Pull the plastic retention pins out of the four corners of the cooler mounting bracket, then remove the cooler mounting bracket.
+
+![CPU cooler mounting bracket](./img/cpu-cooler-mounting-bracket.webp)
+
+4. Push the locking pin outward until it's able to spring away from the motherboard.
+    - Caution: the locking pin may spring up with significant force when freed.
```

Comment:
Would "locking lever" be a better phrase than "locking pin"? It also helps disambiguate from the pins that lock the cooler mounting bracket into place.

Reply:
Sure.